client/src/views/addr.js

```diff
@@ -5,6 +5,7 @@ import search from './search'
 import { txBox } from './tx'
 import { formatSat, formatNumber } from './util'
 import { addrTxsPerPage as perPage, maxMempoolTxs } from '../const'
+import load from '../components/loading'
```

Comment:
I would consider calling this `loader` or `loading` instead of `load`, to make its purpose clearer